src/test/java/seedu/address/model/location/LocationTest.java

```diff
@@ -48,6 +72,35 @@ public void isSameId() {
         assertTrue(ALICE_LOCATION.isSameId(editedAlice));
     }
 
+    @Test
+    public void getPredicateForHighRiskLocations() {
+        // Infected Locations(3) are less than 60% of total locations(7)
+        model.setVisitBook(getWholeVisitBook());
+        Predicate<Location> actualPredicate = Location.getPredicateForHighRiskLocations(model);
+        model.updateFilteredLocationList(actualPredicate);
+        ObservableList<Location> actualList = model.getFilteredLocationList();
+
+        ObservableList<Location> expectedList = FXCollections.observableArrayList();
+        expectedList.add(ALICE_LOCATION);
+        expectedList.add(BENSON_LOCATION);
+        expectedList.add(CARL_LOCATION);
+
+        assertEquals(expectedList, actualList);
+
+        // Infected Locations(6) are more than 60% of total locations(8)
+        model.setVisitBook(getMoreThanSixtyPercentVisitBook());
+        actualPredicate = Location.getPredicateForHighRiskLocations(model);
+        model.updateFilteredLocationList(actualPredicate);
+        actualList = model.getFilteredLocationList();
+
+        expectedList.clear();
+        expectedList.add(ALICE_LOCATION);
+        expectedList.add(BENSON_LOCATION);
+        expectedList.add(CARL_LOCATION);
+
+        assertEquals(expectedList, actualList);
+    }
+
```

Comment:
This test feels like it might be more appropriate elsewhere. This is a class for tests involving location as itself. Perhaps this test might belong to a info handler or for a class that directly implements this.

Reply:
Fixed.